src/app/_components/Links/SocialLinksList.js

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import { getSocialMediaList } from '@components/Links/socialMediaActions';
+
+export function SocialLinksList({ className }) {
+  const [socialMediaList, setSocialMediaList] = useState(null);
+
+  useEffect(() => {
+    getSocialMediaList().then(data => {
+      setSocialMediaList(data.socialMediaList);
+    });
+  }, []);
```

Comment:
Transform this component to a server component

Reply:
Component transformed to a server one